src/js/stores/MesosStateStore.js

```diff
@@ -90,22 +90,33 @@ class MesosStateStore extends GetSetBaseStore {
       .subscribe(this.onStreamData, this.onStreamError);
   }
 
+  getLastMesosState() {
+    const lastState = this.get("lastMesosState") || {};
```

Comment:
This implementation lacks `master_info`. We could do `Object.assign({ <defaults> }, this.get("lastMesosState"))` so we won't need to add every new piece here.